packages/server/src/database/migrations/20200509004247_create-notifications.js

```diff
@@ -0,0 +1,17 @@
+export function up(knex) {
+  return knex.schema.createTable('UserNotifications', function (table) {
```

Comment:
If we want to send a notification more than once (like reminders), how would this work? Would we have different `userNotificationTypeId` for each reminder?